src/bin/nydus-image/main.rs

```diff
@@ -971,6 +982,14 @@ impl Command {
             BootstrapManager::new(Some(bootstrap_path), parent_path)
         };
 
+        // Unsupported conversionTypes has been checked and filtered by `get_batch_size()`.
+        if build_ctx.batch_size > 0 {
```

Comment:
We should only enable small chunk merging for RAFS v6, so need to check fsversion.

Reply:
Changed.
I've add fsversion check here. But I'm not sure if it is better to check fsversion in `get_batch_size()`, or move the ConversionType check from `get_batch_size()` to here. Would you please give me some suggestions?

Reply:
That's better.

Reply:
Sorry, which one is better?

Reply:
Changed.